Harvest.Core/Services/SlothService.cs

```diff
@@ -0,0 +1,210 @@
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Anlab.Core.Models.SlothModels;
+using Harvest.Core.Data;
+using Harvest.Core.Domain;
+using Harvest.Core.Models;
+using Harvest.Core.Models.Settings;
+using Harvest.Core.Models.SlothModels;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using Serilog;
+
+namespace Harvest.Core.Services
+{
+    public interface ISlothService
+    {
+        Task<SlothResponseModel> MoveMoney(Transfer moneyTransfer);
+
+        Task ProcessTransferUpdates();
+    }
+
+    public class SlothService : ISlothService
+    {
+        private readonly AppDbContext _dbContext;
+        private readonly SlothSettings _slothSettings;
+        private readonly IFinancialService _financialService;
+
+        public SlothService(AppDbContext dbContext, IOptions<SlothSettings> slothSettings, IFinancialService financialService)
+        {
+            _dbContext = dbContext;
+            _slothSettings = slothSettings.Value;
+            _financialService = financialService;
+        }
+
+
+
+        //TODO: Add validation?
+        public async Task<SlothResponseModel> MoveMoney(Transfer moneyTransfer)
+        {
+            var token = _slothSettings.ApiKey;
+            var url = _slothSettings.ApiUrl;
+
+            if (string.IsNullOrWhiteSpace(token))
+            {
+                Log.Error("Sloth Token missing");
+            }
+
+            var debit = await _financialService.IsValid(moneyTransfer.FromAccount.Number);
+            var credit = await _financialService.IsValid(moneyTransfer.ToAccount.Number);
```

Comment:
If these debit and credit are invalid, do something with them. Maybe throw an exception?